ui/src/alerting/components/notifications/statusChange.ts

```diff
@@ -0,0 +1,36 @@
+import {omit} from 'lodash'
+import {StatusRuleItem, LevelRule} from 'src/types'
+
+type Changes = 'changes from' | 'equal'
```

Comment:
This is the hill I die on.

```suggestion
type Change = 'changes from' | 'equal'
```

Reply:
hahahah